### PR TITLE
chore: Live playground on Storybook

### PR DESCRIPTION
**Pull request by gabriellsh**
With ability to choose code highlight theme and preview position (side or bottom)

image

image

image

### CODE REVIEW

File: packages/fuselage/src/playground/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import shadesOfPurple from 'prism-react-renderer/themes/shadesOfPurple';
+import synthwave84 from 'prism-react-renderer/themes/synthwave84';
+import ultramin from 'prism-react-renderer/themes/ultramin';
+import vsDark from 'prism-react-renderer/themes/vsDark';
+import * as cssInJs from '@rocket.chat/css-in-js';
+import * as hooks from '@rocket.chat/fuselage-hooks';
+
+import * as components from '..';
+
+const scope = { components, React, cssInJs, hooks };
```
Comment (tassoevan) [inline packages/fuselage/src/playground/index.js:25]: Why not spread all the symbols? I don't think they collide:

```suggestion
const scope = { ...components, ...React, ...cssInJs, ...hooks };
```

File: packages/fuselage/src/playground/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import duotoneLight from 'prism-react-renderer/themes/duotoneLight';
+import github from 'prism-react-renderer/themes/github';
+import nightOwl from 'prism-react-renderer/themes/nightOwl';
+import nightOwlLight from 'prism-react-renderer/themes/nightOwlLight';
+import oceanicNext from 'prism-react-renderer/themes/oceanicNext';
+import palenight from 'prism-react-renderer/themes/palenight';
+import shadesOfPurple from 'prism-react-renderer/themes/shadesOfPurple';
+import synthwave84 from 'prism-react-renderer/themes/synthwave84';
+import ultramin from 'prism-react-renderer/themes/ultramin';
+import vsDark from 'prism-react-renderer/themes/vsDark';
```
Comment (tassoevan) [inline packages/fuselage/src/playground/index.js:19]: Pick just one.

Comment (gabriellsh) [reply packages/fuselage/src/playground/index.js:19]: Sorry if I believe in a world where programmers can express themselves using whatever theme they want...

Jk, I guess I just went a little too overboard with it 😝 .

Comment (tassoevan) [reply packages/fuselage/src/playground/index.js:19]: To be fair we can rollback when Storybook allow us to switch light and dark mode in the future.

**Pull request merged**